README.md

```diff
@@ -1,16 +1,37 @@
 # Bonsai - Machine Learning
 
-The Bonsai.ML project is a collection of packages with reactive infrastructure for adding machine learning algorithms in Bonsai. Below you will find the list of packages (and the included subpackages) currently available within the Bonsai.ML collection.
+The **Bonsai.ML** project is a collection of packages designed to integrate machine learning algorithms within the Bonsai framework. This document provides an overview of the available packages and their functionalities.
```

Comment:
I have been trying to avoid using the word "framework" in favor of "language". Alternatively we can keep the previous wording and say something like "integrate machine learning algorithms with Bonsai".

Reply:
Makes sense, I've updated the README to reflect this